extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,115 @@
+  height: 2em !important;
+  top: 50% !important;
+  left: var(--average) !important;
+  opacity: 0.6 !important;
+  border-radius: 50%;
+  position: absolute;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::after,
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::before {
+  --animation-duration: 3s;
+  transform: scale(0);
+  box-sizing: border-box;
+  content: '';
+  width: 2em;
+  height: 2em;
+  position: absolute;
+  box-shadow: inset 0 0 0 0.05em var(--i-amphtml-interactive-strong-text-color);
+  border-radius: 50%;
+  animation: pulse var(--animation-duration) linear infinite forwards !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::before {
+  animation-delay: calc(var(--animation-duration) / 2) !important;
+}
+
+@keyframes pulse {
+  0% {
+    transform: scale(0.2);
+    opacity: 0;
+    box-shadow: inset 0 0 0 0.6em var(--i-amphtml-interactive-strong-text-color);
+  }
+  10% {
+    opacity: 1;
+  }
+  20% {
+    opacity: 1;
+    box-shadow: inset 0 0 0 0.3em var(--i-amphtml-interactive-strong-text-color);
+  }
+  60% {
+    opacity: 1;
+  }
+  100% {
+    transform: scale(1);
+    opacity: 0;
+    box-shadow: inset 0 0 0 0.01em
+      var(--i-amphtml-interactive-strong-text-color);
+  }
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator {
+  transform: translate(-50%, -50%) !important;
+}
+
+.i-amphtml-story-interactive-slider-average-text {
+  position: absolute !important;
+  transform: translate(-50%, 100%) scale(0) !important;
+  bottom: 180% !important;
+  line-height: 1.5em !important;
+  font-size: 0.8em !important;
+  padding: 0.5em 1em !important;
+  color: black !important;
+  z-index: 1 !important;
+  border-radius: 1.5em !important;
+  box-shadow: 0 0.4em 0.5em -0.25em rgba(0, 0, 0, 0.2),
+    0 0.3em 1.4em 0.25em rgba(0, 0, 0, 0.14),
+    0 0.75em 1em 0.125em rgba(0, 0, 0, 0.14) !important;
+  background-color: white !important;
+}
+
+.i-amphtml-story-interactive-slider-average-text::after {
+  content: '' !important;
+  position: absolute !important;
+  transform: translateX(-50%) rotate(90deg) !important;
+  left: 50% !important;
+  top: 90% !important;
+  border: 0.75em solid !important;
+  border-color: transparent transparent transparent white !important;
+}
+
+.i-amphtml-story-interactive-active.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-text {
+  left: var(--average) !important;
+  animation: average-pop 5s forwards;
+}
+
+@keyframes average-pop {
+  0% {
+    transform: translate(-50%, 100%) scale(0);
+    opacity: 0;
+  }
+  3% {
+    opacity: 1;
+  }
+  6%,
+  94% {
+    transform: translate(-50%, 0) scale(0.9);
+  }
+  97% {
+    opacity: 1;
+  }
+  100% {
+    transform: translate(-50%, 100%) scale(0);
+    opacity: 0;
+  }
+}
```

Comment:
`5s` Feels a bit long to read 2 words. 
We could follow the [material guidelines for "acknowledgement"](https://material.io/design/communication/confirmation-acknowledgement.html#acknowledgement) and do 3 seconds.

The calculated keyframes for average pop would then be:
```
@keyframes average-pop {
  0% {
    transform: translate(-50%, 100%) scale(0);
    opacity: 0;
  }
  5% {
    opacity: 1;
  }
  10%, 90% {
    transform: translate(-50%, 0) scale(0.9);
  }
  95% {
    opacity: 1;
  }
  100% {
    transform: translate(-50%, 100%) scale(0);
    opacity: 0;
  }
}
```

Reply:
![image](https://user-images.githubusercontent.com/22420856/128382215-18ddb600-cab0-490b-940a-c4815dc62309.png)
Seems like toasts on android are 2s for short or 3.5s for long. With the previous keyframes where we'd only show the bubble between 20% and 80% (effectively only having it visible for 60% of the duration) and it was too quick, but we could consider the entering and exiting transition time (0.3s for each) and make the duration of this animation to be `3/3.5s`(visible time) + `0.3*2` (transitions) = `3.6s/4.1s`

Reply:
Having this up 3s feels like plenty of time to me.